src/compiler/cross-team-collaboration.md

```diff
@@ -0,0 +1,73 @@
+issue to be discussed.
+
+Once discussed, a member of the team will comment on the issue with the
+conclusion of the discussion and linking to the relevant Zulip chat.
+
+## ..to be fixed
+If there is an existing working relationship between a member of the requesting
+team and a contributor to the compiler, then the first option that a team has
+for requesting tasks be completed is to ping that contributor and ask if they
+can complete the task. It is recommended that pings take place in public Zulip
+channels so that..
+
+- ..other contributors that have free time have the opportunity to
+  offer their help.
+- ..other compiler team members/leadership can ensure that requests being made
+  are reasonable (see the rest of this section for the types of issues that the
+  compiler team commits to prioritizing on behalf of other teams).
+
+It is worth considering the available bandwidth of the contributor that the
+request is being made of, and whether their areas of expertise in the compiler
+are relevant.
+
+When there is not a appropriate contact in the compiler team to reach out to
+directly, write a comment on a GitHub issue (or create an issue) describing the
+task that needs completed. Teams should nominate issues for the compiler team
+when issues..
+
+- ..are not already tracked by/part of an existing initiative or working group
+  and..
+- ..are blocking/impeding the work of the other team (e.g. a feature or bug
+  preventing the stabilization of something otherwise complete), but..
+- ..aren't absolutely mission-critical - a soundness bug or otherwise critical
+  issue will be prioritized by the [prioritization working
+  group](./prioritization.html) and addressed through the compiler team's other
+  processes for these bugs.
+
+A detailed description of the feature being requested or the bug to be fixed is
+helpful wherever possible (so that the compiler contributor does not need to
+make a guess as to a solution that would solve the problem for the requesting
+team). If a member of the requesting team isn't explicitly listed as the
+point-of-contact for the issue, then the author of the comment will be assumed
+to be the point-of-contact.
+
+Add the `I-compiler-nominated` label to a issue (you can use `@rustbot label
++I-compiler-nominated` to do this).
+
+Once nominated, the issue will be discussed in a upcoming [triage
+meeting](./triage-meeting.html). The compiler team doesn't always get through
+all nominated issues each week, so it can take more than one meeting for your
+issue to be discussed. In the compiler team's discussion, the issue may..
+
+- ..be accepted, in which case it will be assigned to a contributor. Once
+  assigned, a member of the team will work on the issue. If no work is
+  completed after a reasonable time, then re-nominate the issue and the
+  compiler team will find someone else to complete the work.
+- ..or rejected (e.g. due to insufficient bandwidth, other
```

Comment:
when you say "rejected", do you mean "closed"? We do not close issues terribly often during the review of nominated issues...

I'm basically wondering if there is some formal status associated with "rejected" that someone might expect to be formally encoded. My own take is that there is no formality here. We look at the nomination, decide whether we need to take further action, and remove the nomination label. But "rejected" is a strong term that might be read as implying some further action on our part.

Reply:
I've changed this to "not accepted" which is less strong, and noted in both list items that we remove the nomination label, but have otherwise left this point unchanged, which I think is sufficient.